### PR TITLE
Moved vb6 support statement to microsoftdocs/visualstudio

### PR DESCRIPTION
**Pull request by rpetrusha**
# Moved vb6 support statement to microsoftdocs/visualstudio

## Summary
This PR:
- Deletes vb6-support.md, the Visual Basic 6.0 support statement. It's now available at https://docs.microsoft.com/en-us/visualstudio/vb6/vb6-support.
- Redirects to https://docs.microsoft.com/en-us/visualstudio/vb6/vb6-support.

## Suggested Reviewers

@mairaw

### CODE REVIEW

File: .openpublishing.redirection.json [hunk 1]
```diff
@@ -908,6 +908,10 @@
             "source_path": "docs/visual-basic/getting-started/breaking-changes-in-visual-studio-2015.md",
             "redirect_url": "/dotnet/visual-basic/getting-started/breaking-changes-in-visual-studio"
         },
+        {
+            "source_path": "docs/visual-basic/reference/vb6-support.md",
+            "redirect_url": "/visualstudio/vb6/vb6-support"
+        },
```
Comment (mairaw) [inline .openpublishing.redirection.json]: I think you want to add the "redirect_document_id": true here since they're the same topic

**Review by mairaw — APPROVED** [review]
LGTM. Left a small comment/question

File: .openpublishing.redirection.json (hunk 1 above)
Comment (rpetrusha) [reply .openpublishing.redirection.json]: @mairaw, I've added it. I'll merge when the build completes.

**Comment by rpetrusha** [comment]
Closing and reopening to begin new build.

**Pull request merged**